build.gradle.kts

```diff
@@ -49,7 +49,7 @@ allprojects {
 }
 
 group = "exchange.dydx.abacus"
-version = "1.6.55"
+version = "1.7.0"
```

Comment:
nit: not sure if this needs to be a minor bump since we are only adding a new field, shouldn't be a breaking change I think

Reply:
minor bumps are non-breaking changes~ major bumps are breaking changes 👍 

[patches are supposed to be for bugfixes](https://semver.org/)
```
MAJOR version when you make incompatible API changes
MINOR version when you add functionality in a backward compatible manner
PATCH version when you make backward compatible bug fixes
```

i notice we have a habit of using patch for backwards compatible features (which i'm fine with conforming to) but it's technically incorrect 😛